sql/database.sql

```diff
@@ -6423,6 +6455,7 @@ CREATE TABLE `users` (
   `fname` varchar(255) default NULL,
   `mname` varchar(255) default NULL,
   `lname` varchar(255) default NULL,
+  `suffix` varchar(255) default NULL,
```

Comment:
Actually, I noted it is used in your code so will bring it in also via the upgrade script.

TODO for me:
Add this to the sql upgrade script.